src/main/java/edu/hm/hafner/analysis/parser/ProtoLintParser.java

```diff
@@ -0,0 +1,37 @@
+package edu.hm.hafner.analysis.parser;
+
+import java.util.Optional;
+import java.util.regex.Matcher;
+
+import edu.hm.hafner.analysis.Issue;
+import edu.hm.hafner.analysis.IssueBuilder;
+import edu.hm.hafner.analysis.RegexpLineParser;
+import edu.hm.hafner.analysis.Severity;
+
+/**
+ * Parser for ProtoLint.
+ */
+public class ProtoLintParser extends RegexpLineParser {
+
+    private static final long serialVersionUID = -8347619672754062010L;
+
+    private static final String PROTOLINT_PATTERN = "^\\[([^:]+):(\\d+):(\\d+)\\] (.+)$";
```

Comment:
Can you please use named groups? It makes the regexp more readable, see [EclipseMavenParser](https://github.com/jenkinsci/analysis-model/blob/master/src/main/java/edu/hm/hafner/analysis/parser/EclipseMavenParser.java#L29) for an example.

Reply:
Done